Makefile

```diff
@@ -334,6 +337,7 @@ install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
 	done
 
 	$(INSTALL_M) $(build_bindir)/julia* $(DESTDIR)$(bindir)/
+	$(INSTALL_M) $(build_bindir)/embedding* $(DESTDIR)$(bindir)/
```

Comment:
I think it's ok to install the embedding example, but not to a public location with such a generic name. However, it seems that it would be mostly useful to test if installing to a different locally will affect whether the program runs or not so we can add that test after that's possible.